### PR TITLE
bpo-45019: Silence a warning in test_ctypes.

### PR DESCRIPTION
**Pull request by ericsnowcurrently**
This was missed in PR gh-28319.

https://bugs.python.org/issue45019

### CODE REVIEW

File: Lib/ctypes/test/test_values.py [hunk 1]
```diff
@@ -73,15 +73,17 @@ class struct_frozen(Structure):
                 self.assertTrue([entry.code[i] for i in range(abs(entry.size))])
                 # Check the module's package-ness.
                 with import_helper.frozen_modules():
-                    spec = importlib.util.find_spec(modname)
+                    # Hide the message written by the __hello__ module.
+                    with captured_stdout():
+                        spec = importlib.util.find_spec(modname)
```
Comment (gvanrossum) [inline Lib/ctypes/test/test_values.py]: I'm wondering if it wouldn't be useful to just not print anything from the `__hello__` module unless it's invoked as a `__main__` module? That could possibly allow us to remove a lot of captured_stdout() calls. What is this output actually needed for (other than tradition)?

**Review by gvanrossum — APPROVED** [review]
LGTM!

File: Lib/ctypes/test/test_values.py (hunk 1 above)
Comment (ericsnowcurrently) [reply Lib/ctypes/test/test_values.py]: I'm sure we didn't think about it when the module was added.

File: Lib/ctypes/test/test_values.py [hunk 2]
```diff
@@ -73,6 +73,7 @@ class struct_frozen(Structure):
                 self.assertTrue([entry.code[i] for i in range(abs(entry.size))])
                 # Check the module's package-ness.
                 with import_helper.frozen_modules():
+                    # Hide the message written by the __hello__ module.
```
Comment (gvanrossum) [inline Lib/ctypes/test/test_values.py]: You can drop this comment.

**Review by gvanrossum — COMMENTED** [review]
I would change the title of the PR and the commit when it's merged to mention the change to `flag.py/__hello__`. (Also, why are we pulling the data file out of an outdated tooling script? We might as well stuff an actually `__hello__.py` in the stdlib?)

**Review by gvanrossum — APPROVED** [review]
Sorry, meant to approve as well.

**Comment by ericsnowcurrently** [comment]
I'm going to move the `__hello__` stuff to a different PR.

**Pull request merged**